src/map_io/map_version_packet.cc

```diff
@@ -68,9 +68,16 @@ void MapVersionPacket::pre_read(FileSystem& fs, Map* map, bool skip, bool is_pos
 			map->map_version_.map_version_minor = globv.get_safe_int("map_version_minor");
 			uint32_t ts = static_cast<uint32_t>(globv.get_safe_int("map_version_timestamp"));
 			map->map_version_.map_version_timestamp = ts;
-			map->map_version_.needs_widelands_version_after =
-			   globv.get_int("needs_widelands_version_after", 0);
-			map->calculate_needs_widelands_version_after(is_post_one_world);
+
+			if (globv.has_val("minimum_required_widelands_version")) {
+				map->map_version_.minimum_required_widelands_version =
+				   globv.get_safe_string("minimum_required_widelands_version");
+			} else if (globv.has_val("needs_widelands_version_after")) {
+				map->map_version_.minimum_required_widelands_version = "build ";
+				map->map_version_.minimum_required_widelands_version +=
+				   std::to_string(globv.get_safe_int("needs_widelands_version_after") + 1);
```

Comment:
If the map currently has build "21" as version_after, this will result in "build 22" as a requirement, but should probably be "1.0". I haven't checked what this code is used for, when the map is saved in the editor version "1.0" will always be used. Might be strange for the website, if the code is used there?

Reply:
Maps saved with current master still have `needs_widelands_version_after=20` (which the website shows as "build 21 or newer") so this cannot happen unless you manually edited the version file.

Reply:
Hm, okay. The file `version` for the map Wideworld contains the string `needs_widelands_version_after="21"` in current master. But I don't know how the string ended up there.

Reply:
I set that one manually :)

Reply:
Okay, all fine then.